conf/directorylister.service

```diff
@@ -0,0 +1,10 @@
+[Unit]
+Description=Mount directorylister data on install 
+
+[Mount]
+What=__DATA_DIR__/data
```

Comment:
`__DATA_DIR__` doesn't seem to be set in` manifest.toml`. Is data_dir location being used?

Reply:
That's a failed try to make a mount bind using systemd. The file should be deleted